ui/perfherder/alerts/AlertsView.jsx

```diff
@@ -98,20 +122,48 @@ class AlertsView extends React.Component {
     return getStatus(parseInt(validated.status, 10));
   };
 
-  updateFramework = selection => {
-    const { updateParams } = this.props.validated;
-    const { frameworkOptions } = this.state;
-    const framework = frameworkOptions.find(item => item.name === selection);
-    updateParams({ framework: framework.id });
-    this.setState({ framework, bugTemplate: null }, () =>
-      this.fetchAlertSummaries(),
-    );
+  getDefaultFilterText = () => {
+    const { filterText } = this.props.validated;
+    return filterText === undefined || filterText === null ? '' : filterText;
```

Comment:
I would rewrite as `!filterText ? '' : filterText;`, since `undefined`, `null` and `''` are falsy.

Reply:
I was afraid I might run into other falsy values, that should not be considered falsy

Reply:
Fair enough. 
If `filterText` is 0, this code should treat 0 as truthy. 
So you are interested in protecting against `undefined` or `null`.